packages/canary/src/icons/chaos-engineering-icon.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_23_73)">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M6.00007 0.0429077L6.35362 0.396461L7.35362 1.39646C7.54888 1.59172 7.54888 1.90831 7.35362 2.10357C7.15836 2.29883 6.84178 2.29883 6.64651 2.10357L6.50007 1.95712V4.58537C6.67562 4.64741 6.83617 4.74121 6.97482 4.85986L9.24712 3.54795L9.00463 3.48298C8.7379 3.41151 8.57961 3.13734 8.65108 2.8706C8.72255 2.60387 8.99672 2.44558 9.26345 2.51705L10.6295 2.88308L11.1124 3.01249L10.983 3.49545L10.617 4.86147C10.5455 5.12821 10.2714 5.2865 10.0046 5.21503C9.7379 5.14356 9.57961 4.86939 9.65108 4.60266L9.69331 4.44504L7.47505 5.72575C7.49148 5.81467 7.50007 5.90634 7.50007 6.00001C7.50007 6.09369 7.49148 6.18536 7.47505 6.27427L9.69331 7.55499L9.65108 7.39737C9.57961 7.13064 9.7379 6.85647 10.0046 6.785C10.2714 6.71353 10.5455 6.87182 10.617 7.13855L10.983 8.50458L11.1124 8.98754L10.6295 9.11695L9.26345 9.48298C8.99672 9.55445 8.72255 9.39616 8.65108 9.12942C8.57961 8.86269 8.7379 8.58852 9.00463 8.51705L9.24712 8.45208L6.97482 7.14017C6.83617 7.25881 6.67562 7.35261 6.50007 7.41466V10.0429L6.64651 9.89646C6.84178 9.7012 7.15836 9.7012 7.35362 9.89646C7.54888 10.0917 7.54888 10.4083 7.35362 10.6036L6.35362 11.6036L6.00007 11.9571L5.64651 11.6036L4.64651 10.6036C4.45125 10.4083 4.45125 10.0917 4.64651 9.89646C4.84178 9.7012 5.15836 9.7012 5.35362 9.89646L5.50007 10.0429V7.41466C5.3245 7.35261 5.16394 7.2588 5.02528 7.14013L2.75295 8.45206L2.9955 8.51705C3.26224 8.58852 3.42053 8.86269 3.34906 9.12942C3.27759 9.39616 3.00342 9.55445 2.73668 9.48298L1.37066 9.11695L0.887695 8.98754L1.0171 8.50458L1.38313 7.13855C1.4546 6.87182 1.72877 6.71353 1.9955 6.785C2.26224 6.85647 2.42053 7.13064 2.34906 7.39737L2.30684 7.55492L4.52507 6.27422C4.50865 6.18532 4.50007 6.09367 4.50007 6.00001C4.50007 5.90636 4.50865 5.81471 4.52507 5.7258L2.30684 4.44511L2.34906 4.60266C2.42053 4.86939 2.26224 5.14356 1.9955 5.21503C1.72877 5.2865 1.4546 5.12821 1.38313 4.86147L1.0171 3.49545L0.887695 3.01249L1.37066 2.88308L2.73668 2.51705C3.00342 2.44558 3.27759 2.60387 3.34906 2.8706C3.42053 3.13734 3.26224 3.41151 2.9955 3.48298L2.75295 3.54797L5.02528 4.8599C5.16394 4.74123 5.3245 4.64742 5.50007 4.58537V1.95712L5.35362 2.10357C5.15836 2.29883 4.84178 2.29883 4.64651 2.10357C4.45125 1.90831 4.45125 1.59172 4.64651 1.39646L5.64651 0.396461L6.00007 0.0429077Z" fill="#60606C"/>
+</g>
+<defs>
+<clipPath id="clip0_23_73">
+<rect width="12" height="12" fill="white"/>
```

Comment:
should we remove fill colors as well?

Reply:
See comment above. This should not be a merge blocker.